.circleci/config.yml.in

```diff
@@ -263,14 +263,24 @@ jobs:
   prototype_test:
     docker:
       - image: circleci/python:3.7
+    resource_class: xlarge
     steps:
       - run:
           name: Install torch
-          command: pip install --user --progress-bar=off --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+          command: |
+            pip install --user --progress-bar=off numpy
```

Comment:
Installing `numpy` in a pre-step is no longer part of the official installation instructions at https://pytorch.org/get-started/locally/.

```suggestion
```